packages/ui/src/views/repo/pull-request/details/components/conversation/sections/pull-request-changes-section.tsx

```diff
@@ -103,105 +125,77 @@ const PullRequestChangesSection = ({
+      if (!!codeOwnerApprovalEntries?.length && !!codeOwnerPendingEntries?.length)
+        return {
+          icon: <Icon name="circle" className="text-tertiary-background" />,
+          text: 'Some changes were approved by code owners'
+        }
 
-    if (
-      codeOwnerPendingEntries &&
-      codeOwnerApprovalEntries &&
-      codeOwnerApprovalEntries?.length > 0 &&
-      codeOwnerPendingEntries?.length > 0
-    ) {
-      return (
-        <div className="flex items-center gap-x-2">
-          <Icon name="circle" className="text-tertiary-background" />
-          <span className="text-14 text-foreground-1">Some changes were approved by code owners</span>
-        </div>
-      )
-    }
-    if (latestCodeOwnerApprovalArr && latestCodeOwnerApprovalArr?.length > 0 && reqCodeOwnerLatestApproval) {
-      return (
-        <div className="flex items-center gap-x-2">
-          <Icon name="success" className="text-foreground-success" />
-          <span className="text-14 text-foreground-1">Latest changes were approved by code owners</span>
-        </div>
-      )
-    }
-    if (codeOwnerApprovalEntries && codeOwnerApprovalEntries?.length > 0 && reqCodeOwnerApproval) {
-      return (
-        <div className="flex items-center gap-x-2">
-          <Icon name="success" className="text-foreground-success" />
-          <span className="text-14 text-foreground-1">Changes were approved by code owners</span>
-        </div>
-      )
-    }
-    if (codeOwnerApprovalEntries && codeOwnerApprovalEntries?.length > 0) {
-      if (
-        reqCodeOwnerLatestApproval &&
-        minReqLatestApproval &&
-        latestCodeOwnerApprovalArr &&
-        latestCodeOwnerApprovalArr.length < minReqLatestApproval
-      ) {
-        return (
-          <div className="flex items-center gap-x-2">
-            <Icon name="pending-clock" className="text-icons-alert" />
-            <span className="text-14 text-foreground-1">
-              Latest changes are pending approval from required reviewers
-            </span>
-          </div>
+      if (!!latestCodeOwnerApprovalArr?.length && reqCodeOwnerLatestApproval)
+        return {
+          icon: <Icon name="success" className="text-foreground-success" />,
+          text: 'Latest changes were approved by code owners'
+        }
+
+      if (!!codeOwnerApprovalEntries?.length && reqCodeOwnerApproval)
+        return {
+          icon: <Icon name="success" className="text-foreground-success" />,
+          text: 'Changes were approved by code owners'
+        }
+
+      if (codeOwnerApprovalEntries?.length) {
+        if (
+          reqCodeOwnerLatestApproval &&
+          minReqLatestApproval &&
+          latestCodeOwnerApprovalArr &&
+          latestCodeOwnerApprovalArr?.length < minReqLatestApproval
         )
+          return {
+            icon: <Icon name="pending-clock" className="text-icons-alert" />,
+            text: 'Latest changes are pending approval from required reviewers'
+          }
```

Comment:
Can we add brackets on this. For readability, only omit brackets when the return is on the same line as a the condition
```suggestion
        ) {
          return {
            icon: <Icon name="pending-clock" className="text-icons-alert" />,
            text: 'Latest changes are pending approval from required reviewers'
          }
        }
```